src/Swarm/TUI/View.hs

```diff
@@ -415,13 +416,16 @@ robotsListWidget s = viewport RobotsViewport Vertical (hCenter table)
   -- Keep the base and non sytem robot (e.g. no seed)
   isRelevant robot = robot ^. robotID == 0 || not (robot ^. systemRobot)
   -- Keep the robot that are less than 32 unit away from the base
-  isNear robot = distance (realToFrac <$> robot ^. robotLocation) basePos < 32
+  isNear robot = creative || distance (realToFrac <$> robot ^. robotLocation) basePos < 32
```

Comment:
```suggestion
  isCreativeOrNear robot = creative || distance (realToFrac <$> robot ^. robotLocation) basePos < 32
```

Reply:
I would not overcomplicate it - if you are in creative mode then the definition of near simply changes to "anywhere".